tests/test_exceptions.py

```diff
@@ -212,26 +212,33 @@ def test_MD5MismatchError(self):
           actual md5 sum: deadbeef
         """).strip()
 
-    def PackageNotFoundError(self):
-        package = "Groot"
-        exc = PackageNotFoundError(package)
+    def test_PackageNotFoundError(self):
+        package = "Potato"
         with env_var("CONDA_JSON", "yes", reset_context):
+            #SUBDIR/CHANNEL output testing
+            #with env_var("CONDA_SUBDIR", 'osx-32', reset_context):
             with captured() as c, replace_log_streams():
+                #channel_urls = tuple(get_channel_priority_map(context.channels))
+                exc = PackageNotFoundError(package)
                 conda_exception_handler(_raise_helper, exc)
 
         json_obj = json.loads(c.stdout)
         assert not c.stderr
         assert json_obj['exception_type'] == "<class 'conda.exceptions.PackageNotFoundError'>"
         assert json_obj['message'] == text_type(exc)
-        assert json_obj['package_name'] == package
         assert json_obj['error'] == repr(exc)
 
         with env_var("CONDA_JSON", "no", reset_context):
+            #with env_var("CONDA_SUBDIR", 'osx-32', reset_context):
```

Comment:
remove the commented out line here and around like 218-221 above if we don't need them